src/main/java/core/basesyntax/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.impl;
```

Comment:
```suggestion
package core.basesyntax.service.impl;
```

Reply:
move implementation to the impl package into service package